setup.py

```diff
@@ -30,6 +30,7 @@
     },
     include_package_data=True,
     install_requires=[
+        'functools32;python_version<"3"',
```

Comment:
I would rather use the extra_dependencies section to define dependencies for specific python version.
```
extra_dependencies={'python_version<"3"': ["functools32"]}
```
by doing so we would continue to support users based on very old setuptools

Reply:
Fine with me either way. Not sure its going to help with compatibility since I copied this format from jsonschema (a dependency of swagger_spec_validator).